### DISCUSSION TITLE
Untagged releases on Github Releases

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1424] [Q&A] [answered]
I have been able to publish a release using JReleaser Maven Plugin in Github Actions. The JReleaser step publishes a Github Release entry. And as a final step in the GHA (after JReleaser finishes), I push the tag.

This results in a strange situation. The release is associated with the correct tag (`v0.10.0`):

![image](https://github.com/jreleaser/jreleaser/assets/247403/13fe999b-7d63-442f-aa22-c6d5cd90047b)

But the URL of the release says "untagged":

![image](https://github.com/jreleaser/jreleaser/assets/247403/8722a099-84bf-4ce8-abfb-16bb1b60bb4c)

What have I done wrong here?

**Comment by aalmiray** [comment]
By default JReleaser pushes the tag, you don't have to do it yourself.

**Reply by triceo** [reply]
Is there a situation where that would not happen? (I did not disable it.) Because it does appear that I had to push the tag myself, otherwise it was not there.

**Reply by aalmiray** [reply]
JReleaser won't push the tag if `<release><github><skipTag>` is set to `true`. Default is `false`.

**Comment by triceo** [comment, marked as answer]
I filed https://github.com/jreleaser/jreleaser/issues/1425 as I tried again without pushing the tag manually. I am now convinced this is a bug, or at least an unintended behavior.
